packages/express-backend/src/server.js

```diff
@@ -120,6 +120,25 @@ app.get('/users/:userId/items', async (req, res) => {
   res.status(statusCode).send(items || [])
 })
 
+app.get('/users/:userId/items-bought', async (req, res) => {
+  const userId = req.params.userId
+  const items = await Database.findBoughtItemsByUserId(userId)
+  const statusCode = items === undefined ? 500 : 200
+
+  res.status(statusCode).send(items || [])
+})
+
+app.patch('/users/:userId/items/:itemId', async (req, res) => {
```

Comment:
Based on what's written in `ENDPOINTS.md`, it seems like the `userId` in the path refers to the user id of the *receiver*. When authenticating, we will verify that the caller is the *seller*, as the seller is the person who gets to mark the item as sold. So, I think it would be ideal if the user id in the path was the seller id, and the user id of the recipient was passed in the request body.

Reply:
I talked more about this in a comment on `ENDPOINTS.md`